src/components/Common/CaseStudy.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { CardTitle, BodyPrimary } from '../Typography'
+import { Padding } from 'styled-components-spacing'
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+const CaseStudyWrapper = styled(Padding)`
```

Comment:
This isn't great, lets work out where its used and try and remove it. 

This case study file should only deal with CaseStudy, not a wrapper that's just a styled padding component. 

(don't worry I know this is a component you've extracted to `Common/` but lets improve as we go along.

Reply:
Case Study Wrapper is no more.